### PR TITLE
Fixed "Go back to " sentences with title case

### PR DESCRIPTION
**Pull request by divyaamunugama**
### Purpose
Edited "Go back to " sentences with title case

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 1]
```diff
@@ -8483,7 +8483,7 @@ export const console: ConsoleNS = {
                 title: "Roles"
             },
             rolesEdit: {
-                backButton: "Go back to {{type}}",
+                backButton: "Go back to Groups",
```
Comment (minolih) [inline modules/i18n/src/translations/en-US/portals/console.ts:8486]: Why was the placeholder removed here?

Comment (minolih) [reply modules/i18n/src/translations/en-US/portals/console.ts:8486]: Keep the placeholder and change the passing value for the placeholder (to title case) in the places where this message key is used.

File: modules/i18n/src/translations/fr-FR/portals/console.ts [hunk 2]
```diff
@@ -7176,7 +7176,7 @@ export const console: ConsoleNS = {
                 title: "Rôles"
             },
             rolesEdit: {
-                backButton: "Revenir aux {{type}}",
+                backButton: "Revenir aux Groupes",
```
Comment (minolih) [inline modules/i18n/src/translations/fr-FR/portals/console.ts:7179]: Keep the placeholder and change the passing value for the placeholder (to title case) in the places where this message key is used.

Comment (divyaamunugama) [reply modules/i18n/src/translations/fr-FR/portals/console.ts]: Changes made accordingly.

File: modules/i18n/src/translations/en-US/portals/console.ts (hunk 1 above)
Comment (divyaamunugama) [reply modules/i18n/src/translations/en-US/portals/console.ts]: Changes made accordingly

**Pull request closed without merging**